### PR TITLE
add interceptors to sidebar, fix link

### LINKED ISSUE #1987: Zeebe Gateway Interceptor is not in sidebar
The Zeebe Gateway Interceptor page is not part of the sidebar:
Screenshot 2023-04-19 at 10 45 58

The page is accessible via `Zeebe Gateway > Overview > Interceptor Configuration`
Screenshot 2023-04-19 at 10 45 26

### PR DESCRIPTION
**Pull request by akeller**
## Description

Closes #1987! This orphaned page finds a home.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by pepopowitz — DISMISSED** [review]
💪

File: docs/self-managed/zeebe-deployment/zeebe-gateway/interceptors.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ interceptor, you need to provide your gateway with:
   of the interceptor class, e.g. `com.acme.ExampleInterceptor`
 
 Let's continue with the LoggingInterceptor example. We can provide these
-[configurations](configuration/configuration.md)
+[configurations](/self-managed/zeebe-deployment/configuration/configuration.md)
```
Comment (korthout) [inline docs/self-managed/zeebe-deployment/zeebe-gateway/interceptors.md:139]: On [line 7 of this file](https://github.com/camunda/camunda-platform-docs/pull/2248/commits/ca06975690e497629e3993064e448f0af5648f1a#diff-1285b07996b2ad5298774891877cfaf8d2909daa99b30b16b586ee7d3f2ea357R7), we have a block quote stating:

> This functionality is currently only available in Camunda Platform 8 Self-Managed.

❓ Do we still need this?

**Review by korthout — APPROVED** [review]
👏 Nice one @akeller 

LGTM 👍

File: docs/self-managed/zeebe-deployment/zeebe-gateway/interceptors.md (hunk 1 above)
Comment (akeller) [reply docs/self-managed/zeebe-deployment/zeebe-gateway/interceptors.md:139]: Great catch! I will remove it.

**Pull request merged**